src/content/docs/infrastructure/host-integrations/host-integrations-list/github-integration.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: Github integration
+tags:
+  - New Relic integrations
+  - Github integration
+metaDescription: Set up automatic ingestion of your GitHub Dependabot events by using webhook for ongoing ingestion and historical import for past data.
+freshnessValidatedDate: 2024-10-29
+---
+
+
+Set up automatic ingestion of your GitHub Dependabot events by using webhook for ongoing ingestion and historical import for past data.
+
+<img
+  title="Github Integration dashboard"
+  alt="Github Integration dashboard"
+  src="/images/github-integration-screenshot.webp"
+/>
+
+<figcaption>
+After setting up our Github integration, see your data of teams, repos and users right out of the box.
+</figcaption>
+
+
+## Set up the Github integration [#set-up-github]
```

Comment:
Before we provide procedure to perform the actual setup, don't we need to call out prerequisites for the setup? What conditions must a user meet before the user can proceed with the setup? If there are none, please ignore.

Reply:
Added prerequisites.